nodejs/week3/homework-template/src/backend/api/meals.js

```diff
@@ -0,0 +1,132 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/", async (req, res) => {
+  const {
+    meals,
+    maxPrice,
+    availableReservations,
+    title,
+    createdAfter,
+    limit
+  } = req.query;
+
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+    if(meals) {
+      const meals = await knex("meal").select("*");
+      res.json(meals);
+    }
+
+    if(maxPrice) {
+      const getMaxPrice = parseInt(maxPrice);
+      const meals = await knex("meal").select().where("price", "<", getMaxPrice);
```

Comment:
You are not responding with anything here. Ahh you need to say 
```suggestion
      meals = await knex("meal").select().where("price", "<", getMaxPrice);
```
Otherwise you define a new variable